augur/sequence_traits.py

```diff
@@ -163,7 +164,15 @@ def read_in_features(drm_file):
 
     mutPositions = defaultdict(list)
 
-    df = pd.read_csv(drm_file, sep='\t' if drm_file.endswith('.tsv') else ',')
+    df = pd.read_csv(drm_file, sep=None)
+
+    # Validate column names.
+    observed_columns = set(df.columns)
+    required_columns = {"GENE", "SITE", "ALT", "FEATURE"}

```

Comment:
I don't think `GENE` is actually required... see the last two examples in the docstring and the code (commented) below.